### PR TITLE
Configure tooltip visibility

### PR DESCRIPTION
**Pull request by plavemar**
- describe tooltip options to configure visibility

JIRA: RAIL-2929

### CODE REVIEW

File: docs/15_props__chart_config.md [hunk 1]
```diff
@@ -58,7 +58,10 @@ This article describes the options for configuring a chart.
     stackMeasuresToPercent: true, // boolean
     primaryChartType: "column", // string
     secondaryChartType: "area", // string
-    dualAxis: false // boolean
+    dualAxis: false, // boolean
+    tooltip: {
+        enabled: true //boolean
```
Comment (no23reason) [inline docs/15_props__chart_config.md:63]: Nitpick: missing space between // and boolean ;)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/675/ : SUCCESS in 1m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/427/ : SUCCESS in 7s

File: docs/15_props__chart_config.md [hunk 2]
```diff
@@ -367,6 +370,26 @@ import { InsightView } from "@gooddata/sdk-ui-ext";
 />
 ```
 
+## Configure tooltip visibility
+
+* You can configure tooltip visibility for all cases where tooltip is available
+
+* If tooltip visibility is not explicitly configured, it's defaulted to true and meant to use to disable tooltips.
```
Comment (no23reason) [inline docs/15_props__chart_config.md:377]: What about simpler:

> If tooltip visibility is not explicitly configured, tooltip is shown.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-gate-PR400

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-el7/22/ : SUCCESS in 41s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/677/ : SUCCESS in 1m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/97/ : SUCCESS in 34s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/188/ : SUCCESS in 18s

**Pull request merged**